homeassistant/components/mqtt_eventstream.py

```diff
@@ -100,6 +104,9 @@ def _event_receiver(topic, payload, qos):
                 if state:
                     event_data[key] = state
 
+        if ignore_call_service and event_type == EVENT_CALL_SERVICE:
```

Comment:
You know, this was to me a bit of a conscious decision on my part. 

I didn't want to prevent all events from being executed remotely if desired, so that idea here is each instance can decide whether or not to execute remote services.

I am of course happy to go with your implementation. Let me know if you have a preference, otherwise I'll just make the requested change.